docker-compose.yml

```diff
@@ -93,28 +97,25 @@ services:
       - RESOURCEGROUP
       - SECRET_SA_ACCOUNT_NAME
       - STORAGE_ACCOUNT_DOMAIN
-
-      # override
       - ARO_ADOPT_BY_HIVE=true
       - ARO_CHECKOUT_PATH=/app
       - ARO_INSTALL_VIA_HIVE=true
       - HIVE_KUBE_CONFIG_PATH=/app/secrets/aks.kubeconfig
       - KUBECONFIG=/app/secrets/aks.kubeconfig
       - RP_MODE=development
-    expose:
-      - "8443"
     ports:
-      - "127.0.0.1:8443:8443"
+      - "8443:8443"
     healthcheck:
-      test: ["CMD", "curl", "-k", "http://localhost:8443/healthz"]
-      interval: 30s
+      test: ["CMD", "curl", "-k", "-sS", "-f", "https://localhost:8443/healthz/ready"]
+      interval: 90s
       timeout: 30s
-      retries: 3
+      retries: 5
```

Comment:
I see here we're increasing retries but not increasing the timeout, so interested to hear what the difference is with this endpoint

Reply:
I reverted the changes as I had added the longer interval during testing, but it's not necessary. I've pushed the updated changes now.